src/main/resources/com/hcl/appscan/jenkins/plugin/Messages.properties

```diff
@@ -22,6 +22,14 @@ error.threshold.invalid=Invalid entry. Enter an integer value.
 error.malformed.url=Malformed URL {0}.
 error.server.unavailable=AppScan plugin encountered an unexpected error while connecting to AppScan on Cloud.
 error.mobile.analyzer=Incorrect user input. The legacy Mobile Analyzer technology is no longer supported, we recommend using our Static Analyzer scanning for Mobile Applications.
+error.dynamic.AppScan360= Dynamic Analyzer is available for AppScan on Cloud only.
+error.dynamic.analyzer.AppScan360= HCL AppScan task failed: Incorrect user input. Dynamic analysis (DAST) is available in AppScan on Cloud only.
+warning.allow.intervention.AppScan360= Incorrect scan settings: Intervention is available in AppScan on Cloud only.
+error.OSO.AppScan360= HCL AppScan task failed. Incorrect scan settings. Software Composition Analysis is available in AppScan on Cloud only. 
+error.sca.ui= Software Composition Analysis is available in AppScan on Cloud only.
+error.allow.intervention.ui = Scan intervention is available for AppScan on Cloud.
+error.asoc.certificates.ui = Disabling certificates is available for AppScan 360� only.
```

Comment:
I don't think "Disabling certificates" is the correct wording. Can you change the message to indicate that allowing untrusted connections does not apply to ASoC or something along those lines?

Reply:
Okay, Implemented.